features/fetch-crl/el7/config.pan

```diff
@@ -0,0 +1,18 @@
+
+unique template features/fetch-crl/el7/config;
+
+
+include 'components/systemd/config';
+
+'/software/components/systemd/skip/service' = false;
+
+# This is a customization file
+'/software/components/systemd/unit/fetch-crl-boot/file/replace' = false;
+
+# Fix for issue https://bugzilla.redhat.com/show_bug.cgi?id=1630027
+prefix '/software/components/systemd/unit/fetch-crl-boot/file/config/service';
+'RemainAfterExit' = true;
+
+# Enable fetch-crl CRON
+prefix '/software/components/systemd/unit/fetch-crl-cron';
+'state' = 'enabled';
```

Comment:
How about the following? I feel that it might be easier to follow:

```pan
prefix '/software/components/systemd';

'skip/service' = false;

# This is a customization file
'unit/fetch-crl-boot/file/replace' = false;

# Fix for issue https://bugzilla.redhat.com/show_bug.cgi?id=1630027
'unit/fetch-crl-boot/file/config/service/RemainAfterExit' = true;

# Enable fetch-crl CRON
'unit/fetch-crl-cron/state' = 'enabled';
```

You could even use relative prefix statements, but that might be going a bit far!

Reply:
I am less convinced by this one... I tend to be reluctant to use `prefix` as it makes greping through the templates more difficult... Anyway, my next commit makes a counter-proposal!